tests/queries/0_stateless/02421_simple_queries_for_opentelemetry.sh

```diff
@@ -44,6 +44,19 @@ ${CLICKHOUSE_CLIENT} -q "
     ;"
 }
 
+function check_query_settings()
+{
+    ${CLICKHOUSE_CLIENT} -q "
+        SYSTEM FLUSH LOGS;
+        SELECT attribute
```

Comment:
This test will give too big output, better to search for specific query settings.

Reply:
Update test script and reference file as per review comments